tracer/src/Datadog.Trace/AppSec/InstrumentationGatewayBlockingEventArgs.cs

```diff
@@ -0,0 +1,49 @@
+// <copyright file="InstrumentationGatewayBlockingEventArgs.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using Datadog.Trace.AppSec.Transports;
+using Datadog.Trace.AppSec.Transports.Http;
+using Datadog.Trace.Configuration;
+#if NETFRAMEWORK
+using System.Web;
+using System.Web.Routing;
+#endif
```

Comment:
`#else` here may be better?

Reply:
thanks, fixed in https://github.com/DataDog/dd-trace-dotnet/commit/6f5ca731cbee8cfa6f4544e3c705fd37d48d338f